etc/config/crystal.amazon.properties

```diff
@@ -10,12 +10,16 @@ group.crystal.supportsExecute=true
 group.crystal.compilerType=crystal
 group.crystal.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 group.crystal.licenseLink=https://github.com/crystal-lang/crystal/blob/master/LICENSE
-group.crystal.licenseName=Apache 2
-group.crystal.licensePreamble=Copyright 2012-2024 Manas Technology Solutions.
+group.crystal.licenseName=Apache 2.0 with Swift exception
+group.crystal.licensePreamble=Copyright 2012-2025 Manas Technology Solutions.
 
-compiler.crystal1140.semver=1.14.0
-compiler.crystal1140.exe=/opt/compiler-explorer/crystal-1.14.0/bin/crystal
-compiler.crystal1140.cc=/opt/compiler-explorer/gcc-snapshot/bin/gcc
+compiler.crystal1150.semver=1.15.0
+compiler.crystal1150.exe=/opt/compiler-explorer/crystal-1.15.0/bin/crystal
+compiler.crystal1150.cc=/opt/compiler-explorer/gcc-snapshot/bin/gcc
+compiler.crystal1141.semver=1.14.1
```

Comment:
14.1.0 failed to install - you can either update this PR to only include 1.15, or fix the installation for 14 and let me know.